README.md

```diff
@@ -47,9 +47,10 @@ To avoid upgrades for each iteration of `helm`, the `helmfile` executable delega
 
 ## Status
 
-May 2024 Update - We are inviting Helmfile v1 rc testers! Please see the v1 proposal [here](docs/proposals/towards-1.0.md) and the latest rc release in the [releases](https://github.com/helmfile/helmfile/releases) page. Please file feature requests in [Discussions](https://github.com/helmfile/helmfile/discussions) and bugs in [Issues](https://github.com/helmfile/helmfile/issues).
+May 2025 Update
 
-March 2022 Update - The helmfile project has been moved to [helmfile/helmfile](https://github.com/helmfile/helmfile) from the former home `roboll/helmfile`. Please see roboll/helmfile#1824 for more information.
+* Helmfile v1.x has migrated its YAML processing library from goccy/go-yaml to gopkg.in/yaml.v3 as the default and only supported backend. This change simplifies YAML handling and improves compatibility with the broader Go ecosystem.
+* We are excited about the wide adoption for Helmfile v1 users! If you haven't already upgraded, please go over this v1 proposal [here](docs/proposals/towards-1.0.md) and the latest rc release in the [releases](https://github.com/helmfile/helmfile/releases) page. 
```

Comment:
Doesn't this proposal sound a bit confusing now that v1.1 has been released?
Maybe it could be rewritten to clear up the history of helmfile 😅

Reply:
@reneleonhardt Thanks! Does this clarify it for you? https://github.com/helmfile/helmfile/pull/2046

Reply:
Much better at least for me 😄
Small typo: "have been released".